### PR TITLE
Add support for ordering workflow results

### LINKED ISSUE #1430: Control workflow order
From: https://github.com/zooniverse/Panoptes-Front-End/issues/935

### PR DESCRIPTION
**Pull request by edpaget**
Closes #1430

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -355,7 +355,7 @@
     t.boolean  "ouroboros_created",           default: false, index: {name: "index_users_on_ouroboros_created", where: "(ouroboros_created = false)"}
     t.integer  "subject_limit"
     t.boolean  "private_profile",             default: true, index: {name: "index_users_on_private_profile", where: "(private_profile = false)"}
-    t.index name: "users_idx_trgm_login_display_name", using: :gin, expression: "((COALESCE((login)::text, ''::text) || ' '::text) || COALESCE((display_name)::text, ''::text))"
+    t.tsvector "tsv",                         index: {name: "index_users_on_tsv", using: :gin}
```
Comment (camallen) [inline db/schema.rb]: did this change creep in?

**Comment by camallen** [comment]
LGTM

File: db/schema.rb (hunk 1 above)
Comment (edpaget) [inline db/schema.rb]: Yeah I probably didn't drop the db when I ran it.

Comment (edpaget) [inline db/schema.rb]: Should be fixed.

**Pull request merged**